### PR TITLE
solution

### PR DESCRIPTION
**Pull request by liliia-kostenko**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,3 +1,18 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable
+
+
+def cache(func: Callable) -> Callable:
+    cach = {}
```
Comment (vasylhnatiuk) [inline app/main.py:5]: image
cach? change name please

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]
delete useless prints, and use return only one time in wrapper

**Comment by liliia-kostenko** [comment]
@vasylhnatiuk I have made changes. please check

File: app/main.py [hunk 2]
```diff
@@ -1,3 +1,15 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable
+
+
+def cache(func: Callable) -> Callable:
+    cache_dic = {}
```
Comment (vasylhnatiuk) [inline app/main.py:5]:
```suggestion
    cache_dict = {}
```

Comment (vasylhnatiuk) [reply app/main.py:5]: it will be better name

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]

**Comment by liliia-kostenko** [comment]
@vasylhnatiuk 
updated

**Review by vasylhnatiuk — APPROVED** [review]

**Pull request closed without merging**